packages/frontend2/src/env.js

```diff
@@ -45,6 +46,9 @@ export const env = createEnv({
   client: {
     NEXT_PUBLIC_PLAUSIBLE_DOMAIN: z.string().default('localhost'),
     NEXT_PUBLIC_PLAUSIBLE_ENABLED: coerceBoolean.optional(),
+    NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID: z
+      .string()
+      .default('00000000000000000000000000000000'),
```

Comment:
Let's remove the default here and bailout if it's missing in the Asset Risks' layout, at least until asset-risks become production-ready.